src/aero/actuator/ActuatorBulkFAST.C

```diff
@@ -242,6 +245,63 @@ ActuatorBulkFAST::init_epsilon(const ActuatorMetaFAST& actMeta)
   searchRadius_.sync_host();
 }
 
+void
+ActuatorBulkFAST::stk_turbine_search(
+  const ActuatorMeta& actMeta, stk::mesh::BulkData& stkBulk, bool onlyFine /* = false */)
+{
+  //It seems point_centroids includes all actuator points across all turbines. TODO: Is that right?
+  //If so, we need to build a similar list here that will be of size numberofActuators
+  //that includes all hublocations. The search radius also needs to be of the same size 
+  
+  // create bounding sphere and element boxes based on turbine location and search radius
+  auto points = hubLocations_.template view<ActuatorFixedMemSpace>();
```

Comment:
This templated view function is only for dual views where you can get a view on host or device.  The type for ActFixScalarDouble is already a view: https://github.com/Exawind/nalu-wind/blob/83c6597c0b7afd042cb510615af286e9fae30b29/include/aero/actuator/ActuatorTypes.h#L70-L71